src/translations/sv.json

```diff
@@ -88,7 +88,7 @@
   "legend.person.legend": "Privatpersoner, föreningar och företag kan ansöka om båtplats. Av sökande och företag utanför Helsingfors tas en förhöjd årshyra ut.",
   "legend.overview.title": "Sammanfattning",
   "legend.overview.legend": "Kontrollera att uppgifterna stämmer innan du skickar in ansökan.",
-  "legend.thankyou.title": "Tack, din ansökan har skickats!",
+  "legend.thankyou.title": "Tack, din ansökan har skickats!<br/>Du får snart ett bekräftelsemeddelande till den e-postadress du uppgett.<br/><br/>Om du vill ändra eller annullera din ansökan eller undrar om något som gäller ansökan, skicka e-post till venepaikkavaraukset@hel.fi.",
```

Comment:
`<br />` ?

Reply:
Is not allowed?

Reply:
Is it rendered as expect?

Reply:
Ok it doesn't, it was used in another instance I thought it works. I revise!

Reply:
Done, thanks!